micro_dl/cli/curator_script.py

```diff
@@ -0,0 +1,160 @@
+# %% script to generate your ground truth directory for microDL prediction evaluation
+# After inference, the predictions generated are stored as zarr store.
+# Evaluation metrics can be computed by comparison of prediction to human proof read ground truth.
+#
+import numpy as np
+import os
+from PIL import Image
+import imagio as iio
```

Comment:
imageio needs to be in `lit-monai` environment

Reply:
Assuming that `imagio` here means `imageio`, it is already in the env (as required by skimage).

Reply:
```suggestion
import imageio as iio
```